pkg/output/output.go

```diff
@@ -86,6 +85,15 @@ func Print(resource interface{}) error {
 				}
 			}
 		}
+	case "object.Object":
```

Comment:
Should probably have a specific type for this

Reply:
@gdbranco  can you please past the sample output in the PR as well it will be easier to look what output looks like without having to pull the PR . thanks

Reply:
Will update description soon